app/test_main.py

```diff
@@ -1,3 +1,80 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_ages",
+        [
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="get zeros when ages are under 15"
+            ),
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="test when ages are equal 15"
+            ),
+            pytest.param(
+                23,
+                17,
+                [1, 1],
+                id="test when ages are between 15 and 24"
+            ),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="test when ages are equal 24"
+            ),
+            pytest.param(
+                28,
+                28,
+                [3, 2],
+                id="test when ages are equal 28"
+            ),
+            pytest.param(
+                44,
+                44,
+                [7, 6],
+                id="test if function works for third+ year"
+            ),
```

Comment:
Add test cases to test when the input is a negative number, zero, or a large number.